apps/jupyter/extensions/notebook-runner/src/errors/task-errors.ts

```diff
@@ -78,3 +78,25 @@ export class FileSystemError extends TaskError {
     Object.setPrototypeOf(this, FileSystemError.prototype);
   }
 }
+
+export class ExportError extends TaskError {
+  constructor(
+    public readonly reason: string,
```

Comment:
For both messages here: What is the difference between the reason and the message? Do we really need both?